tests/parm/cpld_control_rrtmgp.nml.IN

```diff
@@ -0,0 +1,398 @@
+&atmos_model_nml
```

Comment:
Could this namelist be merged to cpld_control.nml.IN with rrtmgp options turned on?

Reply:
What is the best way for me to get these GP nml options into creating the ORT?
Whatever that is, I will do